pi5/pinctrl_utils.go

```diff
@@ -229,3 +239,45 @@ func setGPIONodePhysAddrHelper(currNodePath string, physAddress uint64, numCAddr
 	currNodePath = parentNodePath
 	return setGPIONodePhysAddrHelper(currNodePath, physAddress, numCAddrCells)
 }
+
+// Creates a virtual Page to access/manipulate memory related to gpiochip data
+func (b *pinctrlpi5) createGPIOVPage(memPath string) error {
+	var err error
+
+	// Open the 'file' you are trying to map. Note: /dev/gpiomem0 is an alias, so it's file length returns 0
```

Comment:
i am not sure it is accurate to say that its an alias, but rather the file represents devices on the board. see this thread for more info: https://stackoverflow.com/questions/16445420/why-is-mmap-returning-a-size-of-zero